util/getopt.h

```diff
@@ -12,6 +12,7 @@
  * -abcfoo		(packed single-character options and an argument)
  * --foo bar		(long option)
  * --foo=bar		(long option and argument separated by '=')
+ * -b -- ignore these   (some arguments followed by a "--" and then anything)
```

Comment:
"This getopt implementation parses options of the following forms... stuff which isn't options and doesn't get parsed"?

I think your intent here is better served by adding a new paragraph:

```
Upon encountering '--', it consumes that argument (by incrementing optind) and
returns NULL to signal the end of option-processing.  Upon encountering any
argument not starting with '-' (except as an argument to an option) it returns NULL
to signal the end of option-processing (without consuming the argument).
```

This is all standard UNIX getopt behaviour but it's probably worth documenting...